### ISSUE TITLE
Cannot view login page when running locally

### ISSUE THREAD

**Issue by neufeldtech** [patternpaul/SHL#9]
Hey @patternpaul,
I was able to get the site up and running locally following your vagrant instructions. However, when I click on the *login* link, I get a runtime exception. Is there some configuration that I am missing?
![image](https://cloud.githubusercontent.com/assets/8252744/18498919/100f603c-7a00-11e6-91eb-e0b40fc0436d.png)

**Comment by patternpaul** [comment]
@neufeldtech Ahhh derp! It's because the example environment configs didn't have the correct encryption key length. I've updated the repo to contain the correct key length as an example. I've also added an additional step that will auto create an admin user to login to the back end. Fixes here.

https://github.com/patternpaul/SHL/commit/0a840501a709b7f20235d0de654dc034cffb6a24

I'd also make sure to pull latest as I've pushed a bunch of changes for admin user cache issues. (pulling to get the above fix will get those fixes as well.). Keep me posted if this resolves everything!
